en/docs/reference/mediators/respond-mediator.md

```diff
@@ -1,24 +1,22 @@
 # Respond Mediator
 
-The **Respond Mediator** stops the processing of the current message and sends the message back to the client as a response.
+The respond mediator stops the processing of the current message and sends the message back to the client as a response.
 
 ## Syntax
 
-The respond token refers to a `<respond/>` element, which is used to stop further processing of a message and send the message back to the client.
-
 ```xml
 <respond/>
 ```
 
 ## Example
 
-Assume that you have a configuration that sends the request to the Stock Quote service and change the response value when the symbol is WSO2 or CRF. Also assume that you want to temporarily change the configuration so that if the symbol is CRF, the ESB profile just sends the message back to the client without sending it to the Stock Quote service or performing any additional processing. To achieve this, you can add the Respond mediator at the beginning of the CRF case as shown below. All the configuration after the Respond mediator is ignored. As a result, the rest of the CRF case configuration is left intact, allowing you to revert to the original behavior in the future by removing the Respond mediator if required.
+In this example API, if the symbol is WSO2 or CRF it sends the request to the Stock Quote service. If the symbol does not meet that criteria, it sends the message back to the client without sending it to the Stock Quote service or performing any additional processing. To achieve this, you can add the respond mediator in the default case of the [switch mediator]({{base_path}}/reference/mediators/switch-mediator).
```

Comment:
Shall we explain this example a bit more for clarity? I think we can use a similar approach to the original text to explain the use case.